x/concentrated-liquidity/swaps_test.go

```diff
@@ -370,23 +510,16 @@ var (
 		//  		4545 -----|----- 5500
 		//  4000 ---------- 4999
 		"two positions with partially overlapping price ranges, not utilizing full liquidity of second position: eth -> usdc": {
-			tokenIn:       sdk.NewCoin("eth", sdk.NewInt(1800000)),
```

Comment:
Note to reviewer: I did any remaining outdated comments from the few cases that _didn't_ break to ensure that all the reference numbers & scripts in this test file remain correct.

I did this in a separate commit in case we want to revert, but I do not think it is worth the time cost of going in and manually amending those scripts as well